frontend/packages/console-shared/src/components/dynamic-form/index.tsx

```diff
@@ -104,6 +105,7 @@ type DynamicFormProps = FormProps<any> & {
   ErrorTemplate?: React.FC<{ errors: string[] }>;
   noActions?: boolean;
   customUISchema?: boolean;
+  onCancel?: () => void;
```

Comment:
The TypeScript type should match the signature of `onClick` which takes an event